.travis.yml

```diff
@@ -1,20 +0,0 @@
-language: php
```

Comment:
Removing the `travis.php.ini` file should be part of this commit IMHO

Reply:
Removed now